src/main/java/tds/exam/results/mappers/TestMapper.java

```diff
@@ -38,4 +51,53 @@ private static String parseBankKeyFromId(final String itemId) {
 
         return itemId.split("-")[0];
     }
+
+    private static long parseLatestAcademicYear(final String academicYearString) {
+        // If this is a range date, pick the latest year
+        if (academicYearString.contains("-")) {
+            return Long.parseLong(academicYearString.split("-")[1]);
+        }
+
+        return Long.parseLong(academicYearString);
+    }
+
+    private static String createGradeStringFromGrades(final List<String> grades) {
```

Comment:
Added this to the javadoc, but here's the rules:

     * Rules:
     *      If no grades are present, return an empty string
     *      If a single grade is present, return it
     *      If the grades is a contiguous range, hyphenate it (i.e. 7, 8, 9 =  7-9)
     *      If the grade range is a standard high school range (i.e. 9-12), return "HS"
     *      Otherwise, return the list of grades comma delimited, in ascending order.
     *      If non-numeric grades are present, they should be listed first